tests/test_genetic_maps.py

```diff
@@ -150,6 +154,37 @@ def test_correct_url(self):
                 gm.download()
         mocked_get.assert_called_once_with(gm.url, filename=unittest.mock.ANY)
 
+    def test_bad_tar_members(self):
+
+        def mock_retrieve_factory():
```

Comment:
This is two separate tests I think --- the code they share is pretty trivial, and it's easier to read as two tests.